framework/src/index.js

```diff
@@ -9,6 +10,7 @@ const validator = require('./controller/helpers/validator');
 module.exports = {
 	Application,
 	version,
+	...samples,
```

Comment:
Would be nice if we keep the sample namespace, may be we have to export more sample data/fixtures in future as well as part of framework.